pre_push.py

```diff
@@ -37,6 +39,12 @@ def run_static():
     Otherwise, it will return statuscode 1
     """
     success = True
+    success &= do_process(
+        [
+            sys.executable,
+            "{dir}/tools/replace_code_block.py".format(dir=current_directory),
```

Comment:
Can you use os.path.join please?

Reply:
I fixed that, checking the commit before push